src/components/Header/Styled/StyledHeaderH1.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const StyledHeaderH1 = styled.h1`
```

Comment:
It is better to name this component as StyledH1 because you manage this component inside your header folder and so it is clear that the component is for your header.